src/cluster.tsx

```diff
@@ -150,12 +154,46 @@ export default class Cluster extends React.Component<Props, State> {
       .map((leave: Feature) => this.featureClusterMap.get(leave));
   };
 
+  public zoomToClusterBounds = (event: React.MouseEvent<HTMLElement>) => {
+    const markers = Array.prototype.slice.call(event.currentTarget.children);
+    const marker = this.findMarkerElement(
+      event.currentTarget,
+      event.target as HTMLElement
+    );
+    const index = markers.indexOf(marker);
+    const cluster = this.state.clusterPoints[index] as Feature;
+    if (!cluster.properties.cluster_id) {
+      return;
+    }
+    const children = this.state.superC.getLeaves(
+      cluster.properties.cluster_id,
+      Infinity
+    );
+    const childrenBbox = bbox(featureCollection(children));
+    // https://github.com/mapbox/mapbox-gl-js/issues/5249
+    this.context.map.fitBounds(LngLatBounds.convert(childrenBbox as any), {
+      padding: this.props.zoomOnClickPadding!
+    });
+  };
+
+  private findMarkerElement(
+    target: HTMLElement,
+    element: HTMLElement
+  ): HTMLElement {
```

Comment:
Return type should be inferred no?

Reply:
'findMarkerElement' implicitly has return type 'any' because it does not have a return type annotation and is referenced directly or indirectly in one of its return expressions.
:)